app/src/main/res/layout/activity_camera_predefined_animators.xml

```diff
@@ -11,18 +10,31 @@
         android:layout_width="0dp"
         android:layout_height="0dp"
         app:layout_constraintBottom_toTopOf="@id/spinnerView"
-        app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"/>
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
 
     <androidx.appcompat.widget.AppCompatSpinner
         android:id="@+id/spinnerView"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:theme="@style/CameraPredefinedAnimationsActivityTheme"
         android:entries="@array/interpolators_array"
+        android:theme="@style/CameraPredefinedAnimationsActivityTheme"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent" />
 
+    <Button
+        android:id="@+id/buttonCancel"
+        android:visibility="invisible"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="50dp"
+        android:layout_marginBottom="20dp"
+        android:padding="10dp"
+        android:text="Cancel animation"
```

Comment:
Can you define the string in string.xml?

Reply:
Done already, lint fails without it 😄